eurofaang_ri_be/users/models.py

```diff
@@ -36,6 +36,15 @@ def __str__(self):
     organization_country = models.CharField(choices=COUNTRIES_CHOICES, blank=True, null=True)
     role = models.CharField(max_length=2, choices=ROLE_CHOICES, default='PI')
 
+    def save(self, *args, **kwargs):
+        if self.pk is None and self.password:
+            self.set_password(self.password)
+        elif self.pk is not None:
```

Comment:
can this be simplified to just "if else", what happens if self.password is None?